redash/settings/__init__.py

```diff
@@ -222,6 +222,14 @@
     os.environ.get("REDASH_STATIC_ASSETS_PATH", "../client/dist/")
 )
 
+# Time limit for scheduled queries. Set this to -1 to execute without a time limit.
+SCHEDULED_QUERY_TIME_LIMIT = int(
+    os.environ.get("REDASH_SCHEDULED_QUERY_TIME_LIMIT", 3600)
```

Comment:
Now that I look at the existing implementation, I wonder if we should use `-1` as the default to remain backward compatible?

Reply:
+1

Reply:
@jezdez you have queries running longer than an hour?

Reply:
I'm not sure if currently, but it happened in the past. Granted, we're probably not the average user group.

Reply:
I think that an hour default value is definitely reasonable for most cases. I just wonder if we should cause issues for any existing deployments :\

Reply:
Well, I think I change my initial response to +0, since this is a settings we can set it to a larger value before deploy. IOW don't consider our use case as a blocker.

Reply:
I don't see any value in imposing a default limit. As long as it's configurable, a `-1` sounds good.